CMakeLists.txt

```diff
@@ -1500,6 +1500,37 @@ if(ENGINEPRIME)
   endif()
 endif()
 
+# Spleeterpp
+find_package(Spleeter)
+if(Spleeter_Found)
+  message(STATUS "Linking dynamically to existing installation of spleeter")
+  target_include_directories(mixxx-lib PUBLIC ${Spleeter_INCLUDE_DIRS})
+  target_link_libraries(mixxx-lib PUBLIC spleeter::Split)
+else()
+    # Fetch spleeter sources from GitHub and build them statically.
+    message(STATUS "Linking statically to spleeter fetched from GitHub")
+    set(SPLEETER_INSTALL_DIR "${CMAKE_CURRENT_BINARY_DIR}/lib/spleeter-install")
+    ExternalProject_Add(spleeter
+      GIT_REPOSITORY git@github.com:gvne/spleeterpp.git
+      GIT_TAG tags/v0.2.1
```

Comment:
Btw, we can't use v0.2.1 IMHO. We need to use a release that doesn't include https://github.com/gvne/spleeterpp/commit/f2efdb486c8425c9f1e73e81ff42ba858a8bd6fe (e.g. v0.2.0), because otherwise rebuilding Mixxx will become a pain because it redownloads the spleeter models on *every single build*.

Reply:
Thank you! Merged

Reply:
The spleeterpp author said that the redownload issue will be fixed soon with this commit: https://github.com/gvne/spleeterpp/commit/4a4f1e743366e5742db9b137bb4cacec664ab1e1